main/AndroidManifest.xml

```diff
@@ -705,6 +705,20 @@
             android:name="cgeo.geocaching.connector.gc.ImportBookmarkLinks"
             android:exported="true">
 
+            <!-- geocaching.com URL via coord.info redirection -->
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:host="coord.info" />
```

Comment:
Thanks @mortenbruhn for your contribution, very welcomed. 

From a quick test in the emulator it seems a small addition to this code block in `AndroidManifest.xml` is required to make it work:

```
<data android:scheme="http" />
<data android:scheme="https" />
```

Without those lines, a link like `https://www.coord.info/BM/...` in an email did not open in c:geo but in the web browser, whereas with those two lines it opens in c:geo. Tested in the emulator with both GMail and K9 as mail client.

Reply:
Right you are. Done :+1: